tests/integration/targets/iso_customize/aliases

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2022, Ansible Project
+# Copyright (c) 2022, VMware, Inc. All Rights Reserved.
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+shippable/posix/group1
+destructive
+skip/aix
+skip/freebsd
+skip/alpine
+skip/python2.6
+needs/privileged
+needs/root
+skip/rhel7.9
+skip/centos7
```

Comment:
What's the problem with those?

Reply:
It's failed with ansible.posix.mount. It seems a loop device is needed.  

Thanks.

Reply:
I haven't tested, but I reckon the other Linux flavours have some automatic mechanism for mounting regular files as loop devices. However, specifying a loop device in the mount should work with all of them. Worth a try, IMO.

Reply:
Hi Russoz,
Thanks for your suggestion.
The module [ansible.posix.mount](https://docs.ansible.com/ansible/latest/collections/ansible/posix/mount_module.html) have not the parameter to specify the loop device. Using separated command "mount" in specific platform with "when" sentence seem won't make the testcase simple and clear. I have no resource to test it locally.

For testcase, I think the it's best way to skip the platforms which don't support the standard module.
Is that OK for you ?

Thanks very much and have a nice day ! 

Best regards
Yuhua Zou

Reply:
Unfortunately loop devices don't work in Docker containers, so this only works well with VMs (RHEL, macOS, FreeBSD are running in VMs, everything else in Docker containers). I basically found that out (again) yesterday: https://github.com/ansible-collections/community.general/pull/5293#issuecomment-1250261290

You should probably use `skip/docker` instead of all the explicit `skip/...` for platforms that run in Docker though.

Reply:
@felixfontein 
Hi felixfontein,
Thanks for your comments.

From the CI testing result, testcases are passed for most of OSes (such as Ubuntu, Debian) in docker. 
Only individual OS require loop device with command "mount" (such as rhel7.9).

Additional, docker containers are privileged with "needs/privileged" in aliases file.  So maybe it would be OK to avoid using "skip/docker" which will skip all OS having docker.

Have a nice day!

Best regards
Yuhua Zou

Reply:
The presence of `needs/privileged` basically means that the tests are not run in any Docker container in CI since the containers in CI are not run in privileged mode. You can only run it that way locally by adding the `--docker-privileged` flag to ansible-test.